PubNubSwiftChatSDK.xcodeproj/project.pbxproj

```diff
@@ -701,7 +701,7 @@
 					"@loader_path/Frameworks",
 				);
 				MACOSX_DEPLOYMENT_TARGET = 12.0;
-				MARKETING_VERSION = 0.8.0;
+				MARKETING_VERSION = 0.8.1;
```

Comment:
I manually changed all occurrences related to versioning. This won't be no longer present if we introduce automated product release and the `.yaml` file. There's a separate story for that (`CLEN-2232`) to track the overall progress.